CamBench_Cap/src/main/java/org/cambench/cap/mixedsensitivities/inter2path/falsepositive/ecbmode/EcbMode1_meta.yaml

```diff
@@ -5,7 +5,7 @@ api:
   java.security.Cipher
 
 description:
-  A configuration for the AES cipher using the CBC mode is passed to a method in the reachable branch of a conditional 
+  A configuration for the AES cipher using the GCM mode is passed to a method in the reachable branch of a conditional 
```

Comment:
Minor revision of the sentence. 
A configuration for the AES cipher using the GCM mode. 
This configuration is passed to a method in the reachable branch of a conditional statement, where it is used to initialize a cipher object.